server/src/controllers/Events/resolver.ts

```diff
@@ -323,6 +339,26 @@ ${unsubscribe}
       data: eventSponsorInput,
     });
 
+    const tagNames = data.tags
+      .map((tagName) => tagName.trim())
+      .filter((tagName) => tagName);
+    const eventTagsData: Prisma.tagsCreateManyInput[] = tagNames.map(
+      (tagName) => ({ name: tagName }),
+    );
+    await prisma.tags.createMany({ data: eventTagsData, skipDuplicates: true });
+    const tags = await prisma.tags.findMany({
+      where: { name: { in: tagNames } },
+    });
```

Comment:
Could you DRY this?  It's the same sequences of operations as for `createEvent`.